.circleci/config.yml

```diff
@@ -17,7 +17,9 @@ datadog_agent_image: &datadog_agent_image datadog/docker-dd-agent:latest
 test_agent: &test_agent
   - image: kyleverhoog/dd-trace-test-agent:latest
     environment:
+      # Use the cloned snapshot directory (see below)
       - SNAPSHOT_DIR=dd-trace-py/tests/snapshots
+      # Enable CI mode for the test agent
```

Comment:
well, what is `CI` mode? 😆

Reply:
Maybe just worth pointing to test agent documentation when it's available. CI mode is probably going to evolve and not worth duping the information here which would become outdated